get_ind_1x.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+conda init bash
+conda activate env_tools
```

Comment:
I'm a tad nervous about including these scripts unless you provide the conda enviornment you used, as otherwise these shell scritpts currently aren't widely 'portable' for others

(i.e., if a future developer has to take over or you have to regen the test data on another platform, this may not work)